sda-auth/main.go

```diff
@@ -390,6 +390,14 @@ func main() {
 	app.Get("/elixir/login", authHandler.getElixirLogin)
 	app.Get("/elixir/cors_login", authHandler.getElixirCORSLogin)
 
+	// Info endpoint
+	publicKey, err := readPublicKeyFile(authHandler.Config.C4ghPubKey)
+	if err != nil {
+		log.Error("Failure to get public key: ", err)
+	}
+	if publicKey != nil {
+		app.Get("/info", authHandler.getInfo)
+	}
```

Comment:
```suggestion
	app.Get("/info", authHandler.getInfo)
```

The app will have failed before this point since the pubkey is a mandatory config object.